src/repositories/projectRepository.ts

```diff
@@ -296,7 +298,13 @@ export const updateProjectWithVerificationForm = async (
     await ProjectAddress.create({
       title: relatedAddress.title,
       address: relatedAddress.address,
-      networkId: relatedAddress.networkId,
+
+      // Frontend doesn't send networkId for solana addresses so we set it to default solana chain id
+      networkId:
+        relatedAddress.chainType === ChainType.SOLANA
+          ? getDefaultSolanaChainId()
+          : relatedAddress.networkId,
```

Comment:
If it was me, I would move this logic to a method as it is repeated plenty of times.